src/modules/shared/pages/innovation/documents/documents-list.component.ts

```diff
@@ -38,7 +39,7 @@ export class PageInnovationDocumentsListComponent extends CoreComponent implemen
   // Filter
   form = new FormGroup(
     {
-      name: new FormControl('', { validators: [Validators.maxLength(50)], updateOn: 'change' }),
+      name: new FormControl('', { validators: [Validators.maxLength(50)] }),
```

Comment:
Should this be `updateOn: 'blur'` like the others?

Reply:
Same case as the other!